.github/workflows/windows-tests.yml

```diff
@@ -11,7 +10,7 @@ jobs:
       matrix:
         # Prefect only tests 3.9 on Windows
         python-version:
-          - "3.9"
+          - "3.10"
```

Comment:
We might want to keep this 3.9 for the Windows tests. I'm not sure if Prefect is compatible with 3.10 on Windows.

Reply:
👍🏼